### DISCUSSION TITLE
Protein binding to other protein than Albumin or alpha1-glycoprotein

### DISCUSSION THREAD

**Discussion by MSmodeler** [Open-Systems-Pharmacology/Forum#913] [Questions & Problems]
Dear all, 

Im working on a model for endogenous secretion and distribution of sex steroids in man. The main sex steroids of interest (Testosteron, DHT and Estradiol) are mainly and stronger bound to Sexual hormon binding globulin (SHBG) than to Albumin and there is a belief that the Albumin bound fraction is also biolicigal active. However when I set the fu to the reported literature values of total unbound sex steroids (Neither bind to Albumin nor to SHBG) the Simulated values are in good agreement with observed data. But my question is if there a possibility to set up a protein binding  partner different then albumin or alpha1-glycoprotein? I tried to integrate SHBG as a specific binding partner in Molecule building block and set the fu to the value that reflected the absolute unbound + the SHBG bound fraction. Unfortunatly this resulted in to low plasma concentration values of the sex steroids. Is there maybe an opportunity to create SHBG as a compund and then modeling the binding interaction between SHBG and the sex steroids? 

Best wishes and thank you for help!

**Comment by StephanSchaller** [comment]
> Unfortunatly this resulted in to low plasma concentration values of the sex steroids

total or unbound?... if you create SHBG as a protein binding partner (via expression in the individual building block) it will also be relevant to set the concentration there (reference and relative expression value) correctly.

> Is there maybe an opportunity to create SHBG as a compund and then modeling the binding interaction between SHBG and the sex steroids?

Certainly, you create the binding partner as a second compound in PK-Sim and then export the simulation to MoBi and define the binding in MoBi.

Hope this helps.
Stephan

**Reply by MSmodeler** [reply]
Dear Stephan
> total or unbound?

The total concentration. Primary the total fraction is mainly the fraction of interest for me. 

> if you create SHBG as a protein binding partner (via expression in the individual building block) it will also be relevant to set the concentration there (reference and relative expression value) correctly.

Maybe this is the problem. I will have a look if my input values are right. When this is not works I will do the step in MOBI with SHBG as a compund.
Unfortunatly Im not sure if the distinction between these binding partners is really necessary, because Im not sure if Albumin bound sex steroids are really available for metabolism due to weak binding which is my main problem why I want to mention the SHBG binding. Also Im not sure if the change of the fu parameter has influence on the partition coefficents as I red in other discussions. This also would influence the performance of the model.  

But first Thank you for your help!

**Reply by MSmodeler** [reply]
Dear @StephanSchaller,

I have tried to take a look if integration of binding to SHBG is really necessary for my model. So I tried to build a simple model only with the reported clearence values found in literature. As I suggested from the user manual of OSP suite, PK Sim does not take into account how strong a compound is bound to plasma proteins but only if it is bound. Therefore I set the fraction unbound value to the percentage of Albumine bound and unbound T and the binding  partner to unknown (reflecting SHBG as binding partner). My assumption was now that if weakly to Albumin bound Testosteron is bioavilable and therefore also can be metabolized, the whole plasma clearence should result in observed plasma concentration values. But indeed the concentration were lower than the observed values. After setting the fu value to 2 % which reflects the "real" unbound fraction of Testosteron the plasma concentration data were in good agreement with the observed data. In conclusion I suggested that the "real" unbound fraction is the better input parameter for my model. Would you agree with this assumption?
